draft-packages/skirt/README.mdx

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Would be great to grab a designers 2c on this documentation. _Usually_ (and I don't agree that this is the correct thing to do) website documentation = primarily designers audience and storybook = primarily engineer audience

Reply:
I wonder if we could fold this into the page layout documentation. It would be good to get @dlimiter or @katsambarcus' thoughts here too :)

Reply:
I realise most of the skirt stuff is covered in the header docs already in terms of design, so I’ve removed this when consolidating the packages together. I reckon it’s worth thinking about whether we should have a place for some code-usage documentation that sits outside the other documentation, or whether they should all be together.